docs/performance.md

```diff
@@ -3,4 +3,136 @@ uri: "/docs/performance/"
 title: "Performance"
 ---
 
-Add some interesting performance details here.
+# Performance Best Practices
+
+WPGraphQL is designed to be performant, but there are several best practices you can follow to ensure optimal performance in your implementation.
+
+## Query Optimization
+
+### Request Only What You Need
```

Comment:
Wondering if a suggestion to "avoid unnecessary connection nesting" to limit db joins (or similar) works here or is beyond the scope. Practical advice would be to:
- When possible, grab data from the parent object instead of from the connection.
- If you only _sometimes_ need data from the connected object, consider using `@include`/`@defer` directives or splitting your queries up.

Reply:
Ya, I think that makes sense. First goal is to have something on this page rather than the current nothing 😆 

![CleanShot 2024-12-13 at 13 33 43](https://github.com/user-attachments/assets/f03aa9e0-afd5-47e1-8ccd-1b6ca1536e58)

Reply:
> When possible, grab data from the parent object instead of from the connection.

@justlevine do you have a good example of this that you use? I can for sure come up with some examples, just curious if you have any common ones.

Reply:
@justlevine we don't have support for the `@defer` directive, unfortunately.